dipy/viz/ui.py

```diff
@@ -681,12 +681,63 @@ class TextBlock2D(UI):
     Attributes
     ----------
     actor : :class:`vtkTextActor`
-
+        The text actor.
+    message : str
+        The initial text while building the actor.
+    position : (float, float)
+        (x, y) in pixels.
+    color : (float, float, float)
+        RGB: Values must be between 0-1.
+    font_size : int
+        Size of the text font.
+    font_family : str
+        Currently only supports Arial.
+    justification : str
+        left, right or center.
+    bold : bool
+        Makes text bold.
+    italic : bool
+        Makes text italicised.
+    shadow : bool
+        Adds text shadow.
     """
 
-    def __init__(self):
+    def __init__(self, text="Text Block", font_size=18, font_family='Arial',
+                 justification='left', bold=False, italic=False, shadow=False,
+                 color=(1, 1, 1), position=(0, 0, 0)):
```

Comment:
Should be `position=(0, 0)`, but I'm going to approve this PR and change it myself.

Reply:
Ah yes, that's a typo. Sorry for that.